frontend/public/components/dashboard/dashboards-page/overview-dashboard/utilization-card.tsx

```diff
@@ -25,7 +30,7 @@ const getQueries = (flags: FlagsObject) => {
         pluginQueries[queryKey] = pluginQuery.properties.query;
       }
     });
-  return _.defaults(pluginQueries, utilizationQueries);
+  return _.defaults(pluginQueries, utilizationQueries, top5ConsumerQueries);
```

Comment:
there is no need to run `top5ConsumerQueries` unless usage breakdown popup is open and in that case we should only query the ones which are related to the popup. It will result in less remote requests and less rerenders.